src/Orleans.Runtime/Configuration/Options/ResourceOptimizedPlacementOptions.cs

```diff
@@ -0,0 +1,122 @@
+    /// </i></remarks>
+    public float CpuUsageWeight { get; set; } = DEFAULT_CPU_USAGE_WEIGHT;
+
+    /// <summary>
+    /// The default value of <see cref="CpuUsageWeight"/>.
+    /// </summary>
+    public const float DEFAULT_CPU_USAGE_WEIGHT = 0.4f;
+
+    /// <summary>
+    /// The importance of the memory usage by the silo.
+    /// </summary>
+    /// <remarks><i>
+    /// <para>A <u>higher</u> value results in the placement favoring silos with <u>lower</u> memory usage.</para>
+    /// <para>Valid range is [0.00-1.00]</para>
+    /// </i></remarks>
+    public float MemoryUsageWeight { get; set; } = DEFAULT_MEMORY_USAGE_WEIGHT;
+
+    /// <summary>
+    /// The default value of <see cref="MemoryUsageWeight"/>.
+    /// </summary>
+    public const float DEFAULT_MEMORY_USAGE_WEIGHT = 0.3f;
+
+    /// <summary>
+    /// The importance of the available memory to the silo.
+    /// </summary>
+    /// <remarks><i>
+    /// <para>A <u>higher</u> values results in the placement favoring silos with <u>higher</u> available memory.</para>
+    /// <para>Valid range is [0.00-1.00]</para>
+    /// </i></remarks>
+    public float AvailableMemoryWeight { get; set; } = DEFAULT_AVAILABLE_MEMORY_WEIGHT;
+
+    /// <summary>
+    /// The default value of <see cref="AvailableMemoryWeight"/>.
+    /// </summary>
+    public const float DEFAULT_AVAILABLE_MEMORY_WEIGHT = 0.2f;
+
+    /// <summary>
+    /// The importance of the physical memory to the silo.
+    /// </summary>
+    /// <remarks><i>
+    /// <para>A <u>higher</u> values results in the placement favoring silos with <u>higher</u> physical memory.</para>
+    /// <para>This may have an impact in clusters with resources distributed unevenly across silos.</para>
+    /// <para>Valid range is [0.00-1.00]</para>
+    /// </i></remarks>
+    public float PhysicalMemoryWeight { get; set; } = DEFAULT_PHYSICAL_MEMORY_WEIGHT;
+
+    /// <summary>
+    /// The default value of <see cref="PhysicalMemoryWeight"/>.
+    /// </summary>
+    public const float DEFAULT_PHYSICAL_MEMORY_WEIGHT = 0.1f;
+
+    /// <summary>
+    /// The specified margin for which: if two silos (one of them being the local to the current pending activation), have a utilization score that should be considered "the same" within this margin.
+    /// <list type="bullet">
+    /// <item>When this value is 0, then the policy will always favor the silo with the lower resource utilization, even if that silo is remote to the current pending activation.</item>
+    /// <item>When this value is 1, then the policy will always favor the local silo, regardless of its relative utilization score. This policy essentially becomes equivalent to <see cref="PreferLocalPlacement"/>.</item>
+    /// </list>
+    /// </summary>
+    /// <remarks><i>
+    /// <para>Do favor a lower value for this e.g: 0.01-0.05</para>
+    /// <para>Valid range is [0.00-1.00]</para>
+    /// </i></remarks>
+    public float LocalSiloPreferenceMargin { get; set; }
```

Comment:
IMO we should set this to between 5% & 10% by default for the a locality benefit. i.e, biasing towards placing destination activation next to calling activation (that is useful information which placement can leverage).

Reply:
sure

Reply:
forgot to actually set, fixed now, also updated comment to suggest 5-10